src/index.html

```diff
@@ -3,141 +3,174 @@
-  <div class="container">
-
-    <div class="row">
-      <div class="col-md-7 col-xl-6">
-        <div id="code-sample" class="content">
-          <a href="https://dartpad.dartlang.org/fae22cffa7b184b4d27cd96dd633a5af"
-            class='codesample__open-in-dartpad no-automatic-external'
-            target='_blank'>Open in DartPad</a>
-          {% include_relative _main-example.html %}
-        </div>
-      </div>
-      <div class="col-md-5 col-xl-6">
-        <div class="content">
-          <div class="frontpage-slogan-with-footnotes">
-            <div class="frontpage-slogan">
-              <h3>
-                Dart helps you craft beautiful, high-quality experiences across all screens, with:
-                <ul>
-                  <li> A <a tabindex="0" role="button" data-toggle="popover" data-content="Dart started out being optimized for web apps, and is evolving to provide great support for mobile apps. Dart also runs on the command line and server-side.">client-optimized</a>
-                    language </li>
-                  <li> Rich, powerful <a tabindex="0" role="button" data-toggle="popover" data-content="For mobile apps, we recommend <a href='https://flutter.io'>Flutter</a>. For desktop apps, <a href='{{site.webdev}}/angular'>AngularDart</a>.">frameworks</a>
-                    </li>
-                  <li> Delightful, flexible <a tabindex="0" role="button" data-toggle="popover" data-content="These range from <a href='/tools'>general-purpose tools</a> to framework-specific ones like the <a href='https://flutter.io/getting-started'><code>flutter</code> tool</a>.">tooling</a>
-                  </li>
-                </ul>
-                <p>
-                  <a href="/guides/get-started" class="btn btn-primary btn-resizable mr-3">Get started</a>
-                  <a href="/install" class="btn btn-light btn-resizable">Install Dart</a>
-                </p>
-              </h3>
-              <p> <em>Click the
-                  <a tabindex="0" role="button" data-toggle="popover" data-content="In the text above and in the code sample, click dotted underlines to get more details.">underlined
-                    text</a>
-                  to learn more.</em> </p>
-            </div>
-          </div>
-        </div>
-      </div>
+<section class="dash-header">
+    <video autoplay loop muted>
+        <source src="assets/dash/video/hotreload.webm"
+                type='video/webm;codecs="vp8, vorbis"' />
+        <source src="assets/dash/video/hotreload.mp4"
+                type='video/mp4;codecs="avc1.42E01E, mp4a.40.2"' />
+    </video>
+</section>
+<section class="dash-header-callout">
+    <div class="made-by">Made By <span>Google</span></div>
+    <div class="callout-title">Dart is a client-optimized platform with native multi-platform performance and the
+        fastest developer cycle
     </div>
+    <div class="callout-subtitle">Coming from another language? Docs: iOS, Kotlin, JavaScript.</div>
```

Comment:
Comment this out, please. We don't have these docs yet.

Reply:
Yes, the bullets I am going to fix. My main goal with this so far was to get it all functional, styles/padding is straight forward and won't take a lot of time. I'm going to push more stuff up shortly with a lot of these changes done in css

Reply:
commented out!